DFC.ServiceTaxonomy.ApiFunction/CypherQueriesNotes/GetOccupationsWithMatchingSkillsDetailed.md

```diff
@@ -6,67 +6,191 @@
 Here's the current query, formatted and with params replaced by literals...
 
 ```
-MATCH (soc:ncs__SOCCode)-[:ncs__hasSocCode]-(j:ncs__JobProfile)--(o:esco__Occupation )<-[r:esco__isEssentialSkillFor|:esco__isOptionalSkillFor]- (s:esco__Skill ) 
-WHERE s.uri in ['http://data.europa.eu/esco/skill/9436db78-4331-495b-a97d-223fd246de2f'] 
+MATCH (soc:ncs__SOCCode)-[:ncs__hasSocCode]-(j:ncs__JobProfile)--(o:esco__Occupation )<-[r:esco__isEssentialSkillFor|:esco__isOptionalSkillFor]- (s:esco__Skill ),jc:ncs__JobCategory)-[:ncs__hasJobProfile]-(j) 
+WHERE s.uri in ['http://data.europa.eu/esco/skill/9436db78-4331-495b-a97d-223fd246de2f'] WITH soc, o, s, r, collect({name:jc.skos__prefLabel,uri:'testhost.com' + jc.WebsiteURI}) as JobCategories
```

Comment:
Should testhost actually be $apiHost?

Reply:
This is a new app setting that is the public website URL e.g. nationalcareers.service.gov.uk for https://nationalcareers.service.gov.uk/job-categories/creative-and-media